client/src/App.css

```diff
@@ -2,6 +2,7 @@
     width: calc(100% - 350px - 4rem);
     padding: 3.5rem 2rem 0rem 0rem;
     padding-left: calc(350px + 2rem);
+    background-color: var(--bp-primary-bg-color, gray)
```

Comment:
nitpick: Also noticed that this line added was making it so every page was loading with the business panel colour. Not sure why it was added, but I removed it in an upcoming quick fix pr because the colour should only be used when the route loaded is for the business panel.

Reply:
note: Just noticed the reason for it being added to make the text area and comments pop out more on page. Decided to add the article page to the class addition logic for applying the businesspanel colours.